### PR TITLE
TCD-2901: Backtrace - Apache and Sourcemap

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/error-reporting/platform-integrations/apache.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+title: Integrating Apache Traffic Server
+sidebar_label: Apache Traffic Server
+description: Configure Apache Traffic Server to use Backtrace's Invoker as a crash log helper.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document provides instructions on how to configure Apache Traffic Server (ATS) to use Backtrace's Invoker as a crash log helper. The workflow upon ATS crash will be as follows:
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/apache.md:12]:
```suggestion
This document provides instructions on configuring Apache Traffic Server (ATS) to use Backtrace's Invoker as a crash log helper. The workflow upon ATS crash is as follows:
```

File: docs/error-reporting/platform-integrations/apache.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+1. On startup, ATS will launch the invoker, which will monitor the `traffic_server` process.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/apache.md:14]:
```suggestion
1. ATS launches the invoker on startup, which monitors the `traffic_server` process.
```

File: docs/error-reporting/platform-integrations/apache.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+2. Upon `traffic_server` crash, the invoker will launch `ptrace` via a script that provides metadata.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/apache.md:15]:
```suggestion
2. Upon the `traffic_server` crash, the invoker launches `ptrace` via a script that provides metadata.
```

File: docs/error-reporting/platform-integrations/apache.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+mkdir -p ${DM}/
+ptrace --kv="version:$VERSION" $1 -O ${DM}/ats
+if test "$?" == "0"; then
+   coroner -c $CF put -u ${PROJECT} ${TOKEN} ${DM}/*.btt
+
+fi
+```
+
+Ensure that your `traffic_server` processes can execute these scripts and have permission to write to the destination folder specified in `backtrace.sh` (referred to by the variable `DM`).
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/apache.md:72]:
```suggestion
Ensure that your `traffic_server` processes can run these scripts and have permission to write to the destination folder specified in `backtrace.sh` (referred to by the variable `DM`).
```

File: docs/error-reporting/platform-integrations/source-map.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+title: Source Maps Integration Guide - Using backtrace-sourcemap-tools
+sidebar_label: Source Maps
+description: Configure your JS application to automatically upload sourcemap files.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The following steps guides you through configuring your JS application to automatically upload sourcemap files during project build. Additionally, it will explain how to set the appropriate ID in your Backtrace client to match errors with the corresponding sourcemap files.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/source-map.md:12]:
```suggestion
The following steps guide you through configuring your JS application to automatically upload sourcemap files during the project build. Additionally, it explains how to set the appropriate ID in your Backtrace client to match errors with the corresponding sourcemap files.
```

File: docs/error-reporting/platform-integrations/source-map.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    "upload": "https://<universe>.sp.backtrace.io:6098/post?format=sourcemap&token=<token>"
+  }
+}
+```
+
+You need to replace `<universe>` with your universe name, and `<token>` with the symbol upload token generated in your Project Settings under the Symbols section. For on-premise Backtrace servers, replace `<universe>.sp.backtrace.io:6098` with the server and port of your Backtrace instance.
+
+## Step 4: Set Up Automatic Sourcemap Uploads
+
+This method will upload all artifact .map files.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/source-map.md:63]:
```suggestion
This method uploads all artifact .map files.
```

File: docs/error-reporting/platform-integrations/source-map.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+First, append `npm run upload` to the `build` command in your package.json's `scripts` section:
+
+```json
+"scripts": {
+  "build": "my current build command && npm run upload",
+  ...
+}
+```
+
+Finally, add the action script `upload` to execute `backtrace-sourcemap` with `package.json` and the build folder as parameters. In this example, both `package.json` and the `dist` folder are assumed to be located in the project root directory:
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/source-map.md:74]:
```suggestion
Finally, add the action script `upload` to run `backtrace-sourcemap` with `package.json` and the build folder as parameters. In this example, both the `package.json` and the `dist` folder are assumed to be located in the project root directory:
```

**Review by lysannep — APPROVED** [review]
@LoryApiFortress Thank you!

**Pull request merged**